src/lib/button/button.html

```diff
@@ -1,7 +1,6 @@
 <span class="mat-button-wrapper"><ng-content></ng-content></span>
-<div matRipple class="mat-button-ripple"
+<div matRipple class="mat-button-ripple" *ngIf="!_isRippleDisabled()"
```

Comment:
Will this still register the events on the initial render if `_isRippleDisabled` it `false` on start-up?

Reply:
No, the events won't be registered. And the directive won't be initialized either.